shm/Cargo.cpp

```diff
@@ -1,11 +1,19 @@
-#include <Cargo.hpp>
+#include "Cargo.hpp"
 
 Cargo& Cargo::operator+=(const size_t amount) {
-    m_amount += amount;
+    _amount += amount;
     return *this;
 }
 
 Cargo& Cargo::operator-=(const size_t amount) {
-    m_amount -= amount;
+    _amount -= amount;
     return *this;
 }
+
+bool Cargo::operator==(const Cargo& src) const {
+     return (_name == src._name) && (_amount == src._amount) && (_basePrice == src._basePrice);
+}
+
+bool Cargo::operator!=(const Cargo& src) const {
+    return !operator==(src);

```

Comment:
```suggestion
    return !(this == src);
```
In my opinion this is better solution. What do you think?